x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/rest/RestGetRollupIndexCapsAction.java

```diff
@@ -27,7 +28,8 @@ public RestGetRollupIndexCapsAction(Settings settings, RestController controller
     protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) {
         String index = restRequest.param(INDEX.getPreferredName());
         IndicesOptions options = IndicesOptions.fromRequest(restRequest, IndicesOptions.STRICT_EXPAND_OPEN_FORBID_CLOSED);
-        GetRollupIndexCapsAction.Request request = new GetRollupIndexCapsAction.Request(new String[]{index}, options);
+        GetRollupIndexCapsAction.Request request =
+            new GetRollupIndexCapsAction.Request(Strings.commaDelimitedListToStringArray(index), options);
```

Comment:
Can you use `Strings.splitStringByCommaToArray`, it should be equivalent but that's the function used by the other action when they parse the `index` param.